python/tests/test_fs.py

```diff
@@ -29,3 +32,75 @@ def test_read_simple_table_from_remote(s3_localstack):
     table_path = "s3://deltars/simple"
     dt = DeltaTable(table_path)
     assert dt.to_pyarrow_table().equals(pa.table({"id": [5, 7, 9]}))
+
+
+@pytest.mark.s3
```

Comment:
@wjones127 is this you adding an integration test for S3? Based on our other conversations, since it seems like the unit-test were just testing on your local filesystem writing new delta tables.

Reply:
Yes